sdk/data/aztables/test-resources.bicep

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+param baseName string
+
+@description('The principal to assign the role to. This is application object id.')
+param testApplicationOid string
+
+var storageTableContributorRoleId = resourceId(
+  'Microsoft.Authorization/roleDefinitions',
+  '0a9a7e1f-b9d0-4cc4-a60d-0319b160aaa3'
+)
+var location = resourceGroup().location
+var primaryAccountName = '${baseName}prim'
+var encryption = {
+  services: {
+    file: {
+      enabled: true
+    }
+    blob: {
+      enabled: true
+    }
+  }
+  keySource: 'Microsoft.Storage'
+}
+var networkAcls = {
```

Comment:
Are these necessary or copied? Ideally we don't have allow set as default. If possible I would remove the whole net acl block.

Reply:
Oh this is a straight conversion of existing, nvm

Reply:
Yeah, should mention, after talking with Ben, that these apparently get injected into the CI pipeline. There are other ways to do this, and they're probably more self-explanatory. Will do this in a separate PR.

Reply:
Issue filed: https://github.com/Azure/azure-sdk-for-go/issues/23603